trashcli/__init__.py

```diff
@@ -1,3 +1,15 @@
 import os
```

Comment:
Please do not squash the commits when you change the code while the pull request is not ready.
It makes difficult to me understand what you have changed, because the changed will be lost during the squashes.
If you want, you can squash you commits at the end, before the merge in master. I'll tell you when I'm ready

Reply:
OK, I'll remember it at my effort.